serverlessworkflow/modules/ROOT/pages/integrations/custom-functions-knative.adoc

```diff
@@ -99,6 +99,11 @@ Save the Knative service name (`custom-function-knative-service`) to use it in t
 <1> Function's name
 <2> The payload that should be sent in the request
 
+[NOTE]
+====
+You can also send the argument as a CloudEvent. See. xref:core/custom-functions-support.adoc#sending-cloudevents[Sending CloudEvents].
+====
+
```

Comment:
Instead, please create a section here and add this paragraph. For example

```asciidoc
== Sending as CloudEvent

Knative functions support xref:{link to Knative doc}[CloudEvent as the message protocol]. {product_name} can create and post CloudEvent messages in `functionRef`. For more information see xref:core/custom-functions-support.adoc#sending-cloudevents[]
```
You don't need to add anything between `[]`. Asciidoc will use the title in the given link as text.

Reply:
Done.